libsplinter/src/node_id/store/diesel/mod.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Diesel based NodeIdStore
+
+mod operations;
+mod schema;
+use operations::NodeIdOperator;
+
+use super::error::NodeIdStoreError;
+use super::NodeIdStore;
+use diesel::r2d2::{ConnectionManager, Pool};
+use diesel::{Insertable, Queryable};
+use schema::node_id;
+
+/// Database backed NodeIdStore implementation
+pub struct NodeIdDbStore<Conn: diesel::Connection + 'static> {
```

Comment:
Following the naming convention from the other stores, this should be `DieselNodeIdStore`